pkg/api/org_users.go

```diff
@@ -118,18 +144,24 @@ func (hs *HTTPServer) GetOrgUsers(c *models.ReqContext) response.Response {
 	return response.JSON(200, result)
 }
 
-func (hs *HTTPServer) getOrgUsersHelper(ctx context.Context, query *models.GetOrgUsersQuery, signedInUser *models.SignedInUser) ([]*models.OrgUserDTO, error) {
-	if err := hs.SQLStore.GetOrgUsers(ctx, query); err != nil {
+func (hs *HTTPServer) getOrgUsersHelper(c *models.ReqContext, query *dtos.GetOrgUsersQuery, signedInUser *models.SignedInUser) ([]*dtos.OrgUserDTO, error) {
+	if err := hs.SQLStore.GetOrgUsers(c.Req.Context(), query); err != nil {
 		return nil, err
 	}
 
-	filteredUsers := make([]*models.OrgUserDTO, 0, len(query.Result))
+	filteredUsers := make([]*dtos.OrgUserDTO, 0, len(query.Result))
 	for _, user := range query.Result {
 		if dtos.IsHiddenUser(user.Login, signedInUser, hs.Cfg) {
 			continue
 		}
 		user.AvatarUrl = dtos.GetGravatarUrl(user.Email)
 
+		var errAC error
+		user.AccessControl, errAC = hs.getUserAccessControlMetadata(c, user.UserId)
+		if errAC != nil {
```

Comment:
Might be a bit simplier
```suggestion
		if user.AccessControl, errAC := hs.getUserAccessControlMetadata(c, user.UserId); errAC != nil {
```

Reply:
Resolved in #43362